docs/reference-implementation/Managed-prometheus.md

```diff
@@ -0,0 +1,43 @@
+# Managed prometheus in Azure Mission-critical (preview)
```

Comment:
```suggestion
# Managed Prometheus in Azure Mission-critical
```

Reply:
I wonder whether we actually should mention/call it Azure Monitor Workspace (with managed Prometheus) or something?